src/Servers/Kestrel/Core/src/Internal/Http2/Http2Connection.Generated.tt

```diff
@@ -0,0 +1,67 @@
+﻿<#@ template debug="false" hostspecific="false" language="C#" #>
+<#@ assembly name="System.Core" #>
+<#@ assembly name="$(NuGetPackageRoot)\microsoft.net.http.headers\2.1.0\lib\netstandard2.0\Microsoft.Net.Http.Headers.dll" #>
+<#@ import namespace="System.Collections.Generic" #>
+<#@ import namespace="Microsoft.Net.Http.Headers" #>
+<#@ output extension=".cs" #>
+<#
+    static IEnumerable<(string Name, string Value)> GetStrings()
+    {
+        yield return ("ClientPreface", "PRI * HTTP/2.0\r\n\r\nSM\r\n\r\n");
```

Comment:
Just want to double-check (with https://github.com/aspnet/AspNetCore/issues/14139 in mind): is here `HTTP/2.0` correct? 

Found https://http2.github.io/http2-spec/#ConnectionHeader, which states it is correct, but is this the latest version?

Reply:
That's the latest spec. Even though HTTP/2 is referred to as "HTTP/2" instead of "HTTP/2.0" in most places, the client connection preface is the exception.